### PR TITLE
Replace the Logo of Layer5 with Meshery in config.yml workflow

### LINKED ISSUE #7526: Replace the Logo of Layer5 with Meshery in config.yml workflow
### Description

Currently, we are using the Logo of Layer5 instead of Meshery. This repository is under the organization Meshery, so, there should be the image of Meshery.

**How to resolve this?**
- Go to .github and replace this welcome image with the new one: 

- Make sure to display the images conditionally, based on the theme selected by user.
Here's the sample code:

```
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/GiorgosXou/Random-stuff/main/Programming/StackOverflow/Answers/70200610_11465149/w.png">
  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/GiorgosXou/Random-stuff/main/Programming/StackOverflow/Answers/70200610_11465149/b.png">
  <img alt="Meshery Logo" src="">
</picture>
```

New images are:

Meshery-celebration-Light
Meshery-celebration-Dark

- Download these images, and upload with an appropriate name. Make sure to use conditional rendering as described above.

- After uploading the image, and removing the previous image, go to .github/config.yml folder and update the reference to previous image to the new image in the following part:

```
firstPRMergeComment: >
  Thanks for your contribution to the Layer5 and Meshery community! :tada:
   
  ![Congrats!](https://raw.githubusercontent.com/meshery/meshery.io/master/.github/welcome/Layer5-celebration.png)
  &nbsp; &#9; &nbsp; &#9;  &nbsp; &#9;  &nbsp; &#9; :star: Please leave a [star on the project](../stargazers). :smile:
```
Here, replace ```Congrats!``` with the code
```
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/GiorgosXou/Random-stuff/main/Programming/StackOverflow/Answers/70200610_11465149/w.png">
  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/GiorgosXou/Random-stuff/main/Programming/StackOverflow/Answers/70200610_11465149/b.png">
  <img alt="Shows a black logo in light color mode and a white one in dark color mode." src="https://user-images.githubusercontent.com/25423296/163456779-a8556205-d0a5-45e2-ac17-42d089e3c3f8.png">
</picture>
```
make sure to change the image sources with the path to image in the folder where you upload it.

- That's it.

---
#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by anshgoyalevil**
**Notes for Reviewers**

This PR fixes #7526 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
@goldfishdolphin for your 👀s.

**Review by leecalcote — APPROVED** [review]
Good work, @anshgoyalevil 👏

File: .github/config.yml [hunk 1]
```diff
@@ -17,7 +17,11 @@ newPRWelcomeComment: >
 firstPRMergeComment: >
   Thanks for your contribution to the Layer5 community! :tada:
    
-  ![Congrats!](https://raw.githubusercontent.com/meshery/meshery/master/.github/assets/images/Layer5-celebration.png)
+  <picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/anshgoyalevil/meshery/fb071249e9ecde9d936f8459c6e3ea626b22bb19/.github/assets/images/Meshery-celebration-Dark.svg">
+  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/anshgoyalevil/meshery/fb071249e9ecde9d936f8459c6e3ea626b22bb19/.github/assets/images/Meshery-celebration-Light.svg">
```
Comment (leecalcote) [inline .github/config.yml:22]: @anshgoyalevil this should be at master, not a specific commit. // @ShivangShandilya @goldfishdolphin @vishalvivekm

Comment (anshgoyalevil) [reply .github/config.yml:22]: @leecalcote 

Apologies for that Sir.

I have created a sister PR https://github.com/meshery/meshery/pull/7586

**Pull request merged**